Author: Matt Wheeler <m@funkyhat.org>
Description: Add accelerator keys to common functions
 Add accelerator keys for common operations in gpaint
Bug-Debian: http://bugs.debian.org/550305
Bug-Ubuntu: https://bugs.edge.launchpad.net/ubuntu/+source/gpaint/+bug/444750
Bug: https://savannah.gnu.org/bugs/?27618
Forwarded: https://savannah.gnu.org/patch/?7053

Index: b/gpaint.glade
===================================================================
--- a/gpaint.glade	2009-12-19 17:12:11.000000000 -0200
+++ b/gpaint.glade	2009-12-19 17:12:11.000000000 -0200
@@ -341,6 +341,7 @@
 			  <property name="label" translatable="yes">_New</property>
 			  <property name="use_underline">True</property>
 			  <signal name="activate" handler="on_new_file_menu_activate"/>
+                          <accelerator key="N" modifiers="GDK_CONTROL_MASK" signal="activate"/>
 
 			  <child internal-child="image">
 			    <widget class="GtkImage" id="image25">
@@ -362,6 +363,7 @@
 			  <property name="label" translatable="yes">_Open</property>
 			  <property name="use_underline">True</property>
 			  <signal name="activate" handler="on_open_menu_activate"/>
+                          <accelerator key="O" modifiers="GDK_CONTROL_MASK" signal="activate"/>
 
 			  <child internal-child="image">
 			    <widget class="GtkImage" id="image26">
@@ -383,6 +385,7 @@
 			  <property name="label" translatable="yes">_Save</property>
 			  <property name="use_underline">True</property>
 			  <signal name="activate" handler="on_save_menu_activate"/>
+                          <accelerator key="S" modifiers="GDK_CONTROL_MASK" signal="activate"/>
 
 			  <child internal-child="image">
 			    <widget class="GtkImage" id="image27">
@@ -453,6 +456,7 @@
 			  <property name="label" translatable="yes">_Print</property>
 			  <property name="use_underline">True</property>
 			  <signal name="activate" handler="on_print_menu_activate"/>
+                          <accelerator key="P" modifiers="GDK_CONTROL_MASK" signal="activate"/>
 
 			  <child internal-child="image">
 			    <widget class="GtkImage" id="image30">
@@ -480,6 +484,7 @@
 			  <property name="label" translatable="yes">_Close</property>
 			  <property name="use_underline">True</property>
 			  <signal name="activate" handler="on_close_window_activate"/>
+                          <accelerator key="W" modifiers="GDK_CONTROL_MASK" signal="activate"/>
 
 			  <child internal-child="image">
 			    <widget class="GtkImage" id="image31">
@@ -501,6 +506,7 @@
 			  <property name="label" translatable="yes">_Quit</property>
 			  <property name="use_underline">True</property>
 			  <signal name="activate" handler="on_quit_menu_activate"/>
+                          <accelerator key="Q" modifiers="GDK_CONTROL_MASK" signal="activate"/>
 
 			  <child internal-child="image">
 			    <widget class="GtkImage" id="image32">
@@ -535,6 +541,7 @@
 			  <property name="label" translatable="yes">Cu_t</property>
 			  <property name="use_underline">True</property>
 			  <signal name="activate" handler="on_cut_menu_activate"/>
+                          <accelerator key="X" modifiers="GDK_CONTROL_MASK" signal="activate"/>
 
 			  <child internal-child="image">
 			    <widget class="GtkImage" id="image33">
@@ -556,6 +563,7 @@
 			  <property name="label" translatable="yes">_Copy</property>
 			  <property name="use_underline">True</property>
 			  <signal name="activate" handler="on_copy_menu_activate"/>
+                          <accelerator key="C" modifiers="GDK_CONTROL_MASK" signal="activate"/>
 
 			  <child internal-child="image">
 			    <widget class="GtkImage" id="image34">
@@ -577,6 +585,7 @@
 			  <property name="label" translatable="yes">_Paste</property>
 			  <property name="use_underline">True</property>
 			  <signal name="activate" handler="on_paste_menu_activate"/>
+                          <accelerator key="V" modifiers="GDK_CONTROL_MASK" signal="activate"/>
 
 			  <child internal-child="image">
 			    <widget class="GtkImage" id="image35">
@@ -625,6 +634,7 @@
 			  <property name="label" translatable="yes">_Select All</property>
 			  <property name="use_underline">True</property>
 			  <signal name="activate" handler="on_select_all_activate"/>
+                          <accelerator key="A" modifiers="GDK_CONTROL_MASK" signal="activate"/>
 			</widget>
 		      </child>
 		    </widget>
